signing_service/signing_service/signing_service.py

```diff
@@ -268,6 +279,23 @@ def _handle_connection(
             # If received frame is correct, validate transaction.
             else:
                 golem_message_response = self._get_signed_transaction(middleman_message.payload)
+                if isinstance(golem_message_response, SignedTransaction):
+                    transaction_sum_combined = self.signing_service_daily_transaction_sum_so_far + middleman_message.payload.value
+                    if transaction_sum_combined > MAXIMUM_DAILY_THRESHOLD:
+                        logger.warning(
+                            f'Signing Service is unable to transact more then {MAXIMUM_DAILY_THRESHOLD} GNTB today.'
+                        )
+                        self.notifier.send()
+                        golem_message_response = TransactionRejected(
+                            reason=TransactionRejected.REASON.DailyLimitExceeded,
+                            nonce=middleman_message.payload.nonce,
+                        )
+                    elif transaction_sum_combined > WARNING_DAILY_THRESHOLD:
+                        logger.warning(f'Signing Service has signed transactions worth {transaction_sum_combined} GNTB today.')
```

Comment:
same as above

Reply:
Above case send reject, where this one pass, only with warning. So as we discuss, I'm leaving it as it is.